src/examples/cpmm_camelot_aave/index.ts

```diff
@@ -2,6 +2,40 @@ import { Backtest } from '../../lib/backtest.js';
 import { DataSourceInfo } from '../../lib/datasource/types.js';
 import { waitFor } from '../../lib/utils/utility.js';
 import { CpmmHedgedStrategy } from './strategy.js';
+import axios, { AxiosResponse } from 'axios';
```

Comment:
the repo uses `graphql-request` so I would give it some thought before adding yet another graphql query library